app-editors/neovim/neovim-0.3.8.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+DESCRIPTION="Vim-fork focused on extensibility and agility."
+HOMEPAGE="https://neovim.io"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/neovim/neovim.git"
+else
+	SRC_URI="https://github.com/neovim/neovim/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~x86"
+fi
+
+LICENSE="Apache-2.0 vim"
+SLOT="0"
+IUSE="+clipboard +luajit +nvimpager python remote ruby +tui +jemalloc"
+
+CDEPEND="
+	dev-libs/libuv:0=
+	dev-libs/msgpack:0=
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? (
+		dev-lang/lua:=
+		dev-lua/LuaBitOp
+	)
+	tui? (
+		dev-libs/libtermkey
+		>=dev-libs/unibilium-2.0.0:0=
+	)
+	<dev-libs/libvterm-0.1
+	dev-lua/lpeg[luajit=]
+	dev-lua/mpack[luajit=]
+	jemalloc? ( dev-libs/jemalloc )
+	net-libs/libnsl"
+
+DEPEND="
+	${CDEPEND}
+	dev-util/gperf
+	virtual/libintl
+	virtual/libiconv
+	app-eselect/eselect-vi"
+
+RDEPEND="
+	${CDEPEND}
+	python? ( dev-python/neovim-python-client )
+	ruby? ( dev-ruby/neovim-ruby-client )
+	remote? ( dev-python/neovim-remote )
+	clipboard? ( || ( x11-misc/xsel x11-misc/xclip ) )"
+
+CMAKE_BUILD_TYPE=Release
+
+src_prepare() {
+	# use our system vim dir
+	sed -e '/^# define SYS_VIMRC_FILE/s|$VIM|'"${EPREFIX}"'/etc/vim|' \
```

Comment:
Use outer `"` and escape `$` when you need. That `'"` transition is really confusing.

Reply:
Done, looks much better now!